jetbrains-core/src/software/aws/toolkits/jetbrains/core/executables/ExecutableManager.kt

```diff
@@ -123,9 +123,11 @@ class DefaultExecutableManager : PersistentStateComponent<ExecutableStateList>,
 
             future.complete(
                 when {
-                    instance is ExecutableWithPath && persisted.autoResolved == true && instance.executablePath.isNewerThan(lastKnownFileTime) ->
+                    // If this is an autoresolved path, and has been updated, remove the autoresolvedness and update
+                    instance is ExecutableInstance.Executable && persisted.autoResolved == true && instance.executablePath.isNewerThan(lastKnownFileTime) ->
```

Comment:
what happens to executeables that have bene resolved but are invalid?

Reply:
They are attempted to be loaded again with `load`. Hm looking at this again it seems like the real issue is with the second line, so reverting this line change